scripts/generateOrcidFile.js

```diff
@@ -1,8 +1,13 @@
 #!/usr/bin/env node
 "use strict";
 
-const DESTINATION_DIR = "./resources";
-const FILENAME = "orcid_app_info.json";
+if(!process.env.RESOURCEROOT) {
+	console.error('No environment variable RESOURCEROOT found.');
+	process.exit(1);
+}
+
+const DESTINATION_DIR = process.env.RESOURCEROOT;
+const FILENAME = "orcid_public_info.json";
```

Comment:
Just to check, is the client ID in the JSON file actually public info?

Reply:
No, this file has just the authURL.